config/locales/en.yml

```diff
@@ -34,6 +34,10 @@ en:
       # overrides
       observation:
         create: "Add"
+    popover:
+      ncr_amount:
+        title: "Transaction Amount Thresholds"
+        content: "$3,000 for supplies<br>$2,500 for services<br>$2,000 for construction"
```

Comment:
Confirmed these values:
====
$3,500 for supplies
$2,500 for services
$2,000 for construction
====